src/jdk.crypto.mscapi/windows/classes/sun/security/mscapi/CKeyStore.java

```diff
@@ -917,6 +941,17 @@ private native void loadKeysOrCertificateChains(String name,
         String location, boolean readOnlyAccess)
             throws KeyStoreException;
 
+    /**
+     * Load keys and/or certificates from inmemory keystore into Collection.
```

Comment:
into a collection is not enforced by this method's contract

Reply:
this is the same comment as the old code path, the only difference between the two is how the keystore is loaded, all subsequent work is shared and hasn't changed